lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -532,16 +535,29 @@ withSizeHeader16be x = do
                                 Conversions
 -------------------------------------------------------------------------------}
 
-coerceBlock  :: Block -> W.Block Tx
-coerceBlock (Block h msgs) =
-    W.Block coerceHeader coerceMessages
+-- | Convert the Jörmungandr binary format block into a simpler Wallet block.
+convertBlock
+    :: (SlotId -> SlotNo)
+    -- ^ Function to determine absolute slot index from an (epoch, local slot).
+    -> Block -- ^ Deserialized Jörmungandr block.
+    -> W.Block Tx
+convertBlock flatten (Block h msgs) =
+    W.Block convertHeader convertMessages
   where
-    coerceHeader = W.BlockHeader (slot h) (parentHeaderHash h)
-    coerceMessages = msgs >>= \case
+    convertHeader = convertBlockHeader flatten h
+    convertMessages = msgs >>= \case
         Initial _ -> []
         Transaction (tx, _wits) -> return tx
         UnimplementedMessage _ -> []
 
+-- | Convert the Jörmungandr binary format header into a simpler Wallet header
+convertBlockHeader
+    :: (SlotId -> SlotNo)
+    -- ^ Function to determine absolute slot index from an (epoch, local slot).
+    -> BlockHeader -- ^ Deserialized Jörmungandr block header.
+    -> W.BlockHeader
+convertBlockHeader flatten h = W.BlockHeader (flatten (slot h)) (parentHeaderHash h)
```

Comment:
```suggestion
convertBlockHeader flatten h =
    W.BlockHeader (flatten (slot h)) (parentHeaderHash h)
```
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters